src/main.js

```diff
@@ -1,6 +1,8 @@
 const electron = require('electron');
 const path = require('path');
 const { app, Tray, Menu } = electron;
+const { app, globalShortcut } = require('electron');
```

Comment:
You can simplify this by doing one-liner

const electron, { app, Tray, Menu } = require('electron')